pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -116,6 +137,97 @@ type ClusterSpec struct {
 	// any resource that does not tolerate the Taint.
 	// +optional
 	Taints []corev1.Taint `json:"taints,omitempty"`
+
+	// ResourceModels is the list of resource modeling in this cluster. Each modeling quota can be customized by the user.
+	// Modeling name must be one of the following: cpu, memory, storage, ephemeral-storage.
+	// If the user does not define the modeling name and modeling quota, it will be the default model.
+	// The default model index from 0 to 8. Each default model's cpu quota is [2^(index-1), 2^index), 2 <= index <= 8
+	// Each default model's memory quota is also [2^(index + 2), 2^(index + 3)), 2 <= index <= 8, E.g. model 0 is like this:
+	// - index: 0
+	// ranges:
+	//   - name: "cpu"
+	//     min: 0 C
+	//     max: 1 C
+	//   - name: "memory"
+	//     min: 0 GB
+	//     max: 4 GB
```

Comment:
```suggestion
	// - grade: 0
	//   ranges:
	//   - name: "cpu"
	//     min: 0 C
	//     max: 1 C
	//   - name: "memory"
	//     min: 0 GB
	//     max: 4 GB
```

1. rename `index` to `grade`.
2. represent in yaml format

Reply:
Done.